pysindy/optimizers/base.py

```diff
@@ -101,7 +101,7 @@ def _reduce(self):
         """
         raise NotImplementedError
 
-    def fit(self, x_, y, sample_weight=None, **reduce_kws):
+    def fit(self, x_, y, p=None, sample_weight=None, **reduce_kws):
```

Comment:
What is `p`? I don't see it being used in this method.

Reply:
yes, this is a vestige of something I tried a while back. You were right to get rid of this